### ISSUE TITLE
bap-common package not found from opam

### ISSUE THREAD

**Issue by SunghoLee** [BinaryAnalysisPlatform/bap#1587]
Hi all,

I'm trying to write a plugin and install the plugin using the dune build system. However, the dune build script of the README.md fails due to the absence of the bap-common package. The reason is that bap installed from opam does not contain the bap-common package. 

I resolved the problem by pinning the recent bap project cloned from Github, but it would be better to contain the bap-common package in the bap on opam.

Is there a plan to update the bap on opam?

**Comment by ivg** [comment]
Yes, there are plans but there are blockers that prevent them. Right now only the testing opam repository works with dune build  system. And dune build system itself is unofficial and mostly for local development.
